tensornetwork/backends/jax/jitted_functions.py

```diff
@@ -53,8 +53,8 @@ def jax_lanczos(matvec, arguments, init, ncv, neig, landelta, reortho):
       reortho: If `True`, reorthogonalize all krylov vectors at each step. 
         This should be used if `neig>1`.
     Returns:
-      list: Eigen values
-      list: Eigen values
+      jax.numpy.ndarray: Eigenvalues
+      list: Eigenvectors
```

Comment:
Should this really be a list? In NumPy it would be an array whose first axis indexes the eigenvectors. That seems more convenient to me in cases where you want to project data onto an eigenspace etc.

Reply:
that's for consistency with block-sparse backends. There one can't easily concatentate the vectors